test/apisix/Dockerfile.test.apisix.rpm.el8.local

```diff
@@ -0,0 +1,42 @@
+ARG IMAGE_BASE="centos"
```

Comment:
APISIX's CI seems to have nothing to do with centos8, right? Why do you need this file?

Reply:
APISIX build tools CI runs on centos7, centos8,ubi and ubuntu. It tries to build, package, install and run apisix.